security/cert_manager_operator/cert-manager-securing-routes.adoc

```diff
@@ -0,0 +1,21 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cert-manager-securing-routes"]
+= Securing routes with the {cert-manager-operator}
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-securing-routes
+
+toc::[]
+
+In the {product-title}, the route API is extended to provide a configurable option to reference TLS certificates via secrets. With the xref:https://docs.openshift.com/container-platform/4.16/release_notes/ocp-4-16-release-notes.adoc#ocp-4-16-networking-secured-route_release-notes[Creating a route with externally managed certificate] feature enabled, you can minimize errors from manual intervention, streamline the certificate management process, and enable the {product-title} router to promptly serve the referenced certificate.
```

Comment:
This looks like a mix of `xref` syntax and `link` syntax. For `xref`, should be a relative link like `../../release_notes` or for `link` be the the `https://` external URL.

But it might be more trouble than it's worth to try linking directly to the release note, especially if the RN blurb isn't providing any new info that this assembly doesn't already cover (if it does, maybe those bits should be added in the intro here instead), considering that this PR will first need to merge to `main` and then get picked to version-specific branches. For example, an `xref` to 4.16-specific `ocp-4-16-release-notes.adoc` is a little tricker to maintain (tho doable).

If you opt to just drop the `xref`, then maybe something like:
```suggestion
In the {product-title}, the route API is extended to provide a configurable option to reference TLS certificates via secrets. With this Technology Preview feature enabled for creating routes with externally managed certificates, you can minimize errors from manual intervention, streamline the certificate management process, and enable the {product-title} router to promptly serve the referenced certificate.
```

Reply:
Oops, I see now that this `xref` is referring to technically a different feature. However, the comments about mixed use of `xref` and `link` remain. To avoid the issues with keeping up with the different-versioned RN files, I might suggest just linking straight to the main book content about this TP feature instead of the RN, e.g.:
```
In the {product-title}, the route API is extended to provide a configurable option to reference TLS certificates via secrets. With the xref:../../networking/routes/secured-routes.adoc#nw-ingress-route-secret-load-external-cert_secured-routes[Creating a route with externally managed certificate] Technology Preview feature enabled, you can minimize errors from manual intervention, streamline the certificate management process, and enable the {product-title} router to promptly serve the referenced certificate.
```